src/Console/ModelsCommand.php

```diff
@@ -1067,8 +1067,9 @@ protected function getClassNameInDestinationFile(object $model, string $classNam
         $className = trim($className, '\\');
         $writingToExternalFile = !$this->write;
         $classIsNotInExternalFile = $reflection->getName() !== $className;
+        $forceFQCN = $this->laravel['config']->get('ide-helper.force_fqn', false);
```

Comment:
I suggest to stay consistent regarding the abbreviation:
```suggestion
        $forceFQN = $this->laravel['config']->get('ide-helper.force_fqn', false);
```